services/booking-api/helpers/booking.js

```diff
@@ -10,6 +10,7 @@ const URI_RESOURCE = {
   CANCEL: 'booking/cancel',
   GET: 'booking/get',
   SEARCH: 'booking/search',
+  GET_HISTORICAL_ATTENDEES: 'booking/getHistoricalAttendees',
 };
```

Comment:
Easy to missunderstand this general constant if you think CANCEL is eq to the string 'CANCEL'?
Maybe it could be named more precise like CANCEL_BOOKING = '...' etc

Reply:
Do you think so even when the constant name is URI_RESOURCE?